src/Admin/CitizenActionAdmin.php

```diff
@@ -29,6 +33,17 @@ class CitizenActionAdmin extends AbstractAdmin
         '_sort_by' => 'name',
     ];
 
+    /** @var EventDispatcherInterface */
+    private $eventDispatcher;
+    private $referentTagManager;
+
+    public function __construct($code, $class, $baseControllerName, ReferentTagManager $referentTagManager)
+    {
+        parent::__construct($code, $class, $baseControllerName);
+
+        $this->referentTagManager = $referentTagManager;
```

Comment:
To avoid the override of the construct, what do you think to inject also by setter injection the `referentTagManager` like the `eventDispatcher` ?